src/components/LMMedia/LMDocument/types.ts

```diff
@@ -0,0 +1,31 @@
+import {TextStyle, ViewStyle} from 'react-native';
+import {LMIconProps} from '../../../base/LMIcon/types';
+
+// data model for attachmentMeta object inside attachments
+interface AttachmentMeta {
+  entityId: string;
+  format: string;
+  name: string;
+  ogTags: Object;
+  size: number;
+  url: string;
+}
+
+// data model for array of attachments
+interface DocumentItem {
+  attachmentMeta: AttachmentMeta;
+  attachmentType: number;
+}
```

Comment:
move this interface to separate folder and use builder pattern to make these models.